src/System.Runtime.Extensions/src/System.Runtime.Extensions.builds

```diff
@@ -9,14 +9,6 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Runtime.Extensions.csproj">
-      <TargetGroup>netstandard1.5</TargetGroup>
-      <OSGroup>Unix</OSGroup>
-    </Project>
-    <Project Include="System.Runtime.Extensions.csproj">
-      <TargetGroup>netstandard1.5</TargetGroup>
-      <OSGroup>Windows_NT</OSGroup>
-    </Project>
-    <Project Include="System.Runtime.Extensions.csproj">
       <TargetGroup>net462</TargetGroup>
```

Comment:
@ericstj are there cases were we need to manually remove entries from the packageIndex.json file?